app/presenters/entry_presenter.rb

```diff
@@ -1,13 +1,17 @@
 class EntryPresenter
   delegate :title,
-           :summary,
            :path,
            to: :entry
 
   WEBSITE_ROOT = Plek.current.website_root.gsub(/https?:\/\//, '')
 
-  def initialize(entry)
+  def initialize(entry, show_summaries)
     @entry = entry
+    @show_summaries = show_summaries
```

Comment:
Are there some tests that exercise this somewhere? Atom feeds seem like a sensitive thing

Reply:
The atom feeds do not have a great test suite. EntryPresenter does not have its own unit tests - it is tested a bit by the feature tests. I'll add some unit tests